File: NodeJs_TypeScript_JavaScript_Server/Project_NodeJs_VSCode/src/base/common/net/tcp/connection/GeneralTcpConnection.ts
import { TcpCryptoConnection } from './TcpCryptoConnection';

import * as net from "net";

export class GeneralTcpConnection extends TcpCryptoConnection {

  public constructor(socket: net.Socket) {
    super(socket);
    
    //void
  }
  
}

File: CPP_Joc_Windows_Android/SH_Client_Win_Cpp_Cmake/App/src/base/net/tcp/GeneralTcpConnection.cpp
#include "GeneralTcpConnection.h"

using namespace base::Net;

GeneralTcpConnection::GeneralTcpConnection(std::shared_ptr<Socket>& socket)
    : super(socket)
{
    //void
}

GeneralTcpConnection::~GeneralTcpConnection() {
    //void
}
